Lib/ClassLibraryCommon/Blob/CloudBlobContainer.cs

```diff
@@ -2491,7 +2491,7 @@ internal RESTCommand<TimeSpan> BreakLeaseImpl(TimeSpan? breakPeriod, AccessCondi
             int? breakSeconds = null;
             if (breakPeriod.HasValue)
             {
-                CommonUtility.AssertInBounds("breakPeriod", breakPeriod.Value, TimeSpan.Zero, TimeSpan.MaxValue);
+                CommonUtility.AssertInBounds("breakPeriod", breakPeriod.Value, TimeSpan.FromSeconds(Constants.MinimumLeaseDuration), TimeSpan.FromSeconds(Constants.MaximumLeaseDuration));
```

Comment:
You should name one of them MinimumLeaseDuration and the other minimum BreakLeasePeriod. Since the break lease APIs take a break period, not a lease duration, we should have two distinct maximum constants as well like MaximumLeaseDuration and MaximumBreakPeriod for a total of 4 constants